examples/VLCB_1in1out/VLCB_1in1out.ino

```diff
@@ -46,16 +46,16 @@ const byte SWITCH0 = 8;             // VLCB push button switch pin
 VLCB::Configuration modconfig;               // configuration object
 VLCB::CAN2515 can2515;                  // CAN transport object
 VLCB::LEDUserInterface ledUserInterface(LED_GRN, LED_YLW, SWITCH0);
-VLCB::SerialUserInterface serialUserInterface(&can2515);
+VLCB::SerialUserInterface serialUserInterface(&modconfig, &can2515);
+VLCB::CombinedUserInterface combinedUserInterface(&ledUserInterface, &serialUserInterface);
```

Comment:
I think you have your repo out of sync again. The CombinedUserInterface is gone. 
Line 49 is also out of date.